helm/ph-ee-engine/templates/dev-redis.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.redis.enabled -}}
```

Comment:
Do we need to add the redis chart like this? Do t we have any ready made chart available from online providers like bitnami?